src/httpClient.ts

```diff
@@ -125,7 +136,7 @@ class DefaultHTTPClient implements HTTPClient {
     }
```

Comment:
Should we use the `node:zlib` bindings instead of pako to encode the request? ^

Reply:
sure we may as well